src/components/dialogs/limits/limits-constants.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export enum LimitsPropertyName {
+    LIMITS_TYPE = 'Limit type',
+}
+
+export function getPropertyAvatar(type: string): string {
+    const transformedType: LimitsPropertyName | undefined = type as LimitsPropertyName;
```

Comment:
I don't understand this line, why it could be undefined if you say right after it's not ?